copier/user_data.py

```diff
@@ -193,6 +198,7 @@ class Question:
     placeholder: str = ""
     secret: bool = False
     type: str = ""
+    validation: Optional[str] = None
```

Comment:
You can just use this:

```suggestion
    validation: str = ""
```

Makes it easier to reason about the value, because it will always be a `str`. Just it will be `""` if the user supplies nothing.